### ISSUE TITLE
Deploy current version to mavencentral

### ISSUE THREAD

**Issue by dasheck0** [SpecialCyCi/AndroidResideMenu#87]
Hi,

I wanted to use your library in a project and it all worked out besides some minor issues. I realised that the current version on mavencentral is rather old (deployed 10 months ago). So unfortunately I cannot use custom menus, since this is not included in the deployed version. Can you make this available for us since I do not want to add it as library module. I did it anyway so that I can use it. But is not the preferred way.

Best regards
dasheck

**Comment by Cartman34** [comment]
You are totally right !! We really need this ! We should not use your lib by another way than gradle !
